client/rest-high-level/src/main/java/org/elasticsearch/client/watcher/WatcherStatsRequest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.client.watcher;
+
+import org.elasticsearch.client.Validatable;
+
+/**
+ * A request to explicitly acknowledge a watch.
+ */
+public class WatcherStatsRequest implements Validatable {
+
+    private boolean includeCurrentWatches;
```

Comment:
if there is no reason to make these mutable, lets make these final and have 2 constructors, a default one where we set them both to (im going to guess false, but thats up to you), and a 2 arg constructor where we ensure both are set.